### ISSUE TITLE
Some UI elements are not translatable

### ISSUE THREAD

**Issue by radimous** [PrismLauncher/PrismLauncher#159]
### Version of PolyMC

Prism 5.0

### Version of Qt

Qt 6.4.0

### Description of bug

Buttons and title in settings can't be translated.
See image below ↓
![image](https://user-images.githubusercontent.com/56518192/196876764-bdf0f817-b895-4dba-8fd8-076d9ea00b5f.png)

### Steps to reproduce

Open settings with any 100% language that isn't english.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by DioEgizio** [comment]
This was also reported in polymc (https://github.com/PolyMC/PolyMC/issues/617)

I think this might be a bug in qt or smth

**Comment by Scrumplex** [comment]
Yes. If you restart the launcher, it's fine. That's an issue with Qt, and we haven't looked into fixing this yet.

**Comment by DioEgizio** [comment]
It happens even after restarting the launcher for me, so idk

**Comment by aipiao-gh** [comment]
Launcher on my computer had a similar issue, where when I selected Chinese, the text in the launcher was half in Chinese and half in English
OS: Windows 11

<img width="1120" height="926" alt="Image" src="https://github.com/user-attachments/assets/a9597f68-5346-4b02-9288-780f71e6915f" />

<img width="1120" height="926" alt="Image" src="https://github.com/user-attachments/assets/96e1b3a5-a913-42ce-bcb0-cd13509e55a0" />

**Comment by DioEgizio** [comment]
> Launcher on my computer had a similar issue, where when I selected Chinese, the text in the launcher was half in Chinese and half in English
> OS: Windows 11
> 
> <img width="1120" height="926" alt="Image" src="https://github.com/user-attachments/assets/a9597f68-5346-4b02-9288-780f71e6915f" />
> 
> <img width="1120" height="926" alt="Image" src="https://github.com/user-attachments/assets/96e1b3a5-a913-42ce-bcb0-cd13509e55a0" />

completely unrelated, see https://github.com/PrismLauncher/Translations/issues/34

**Comment by Trial97** [comment]
linking https://github.com/PrismLauncher/Translations/issues/30 and https://github.com/PrismLauncher/Translations/pull/19 as is relevant to this

**Comment by radimous** [comment]
This issue is partially fixed in Prism 10.0.1
The buttons are translated unless you switch the language. In that case they are in english until you reopen the settings dialog.
Should this issue be closed?

First time opening settings
<img width="755" height="748" alt="Image" src="https://github.com/user-attachments/assets/43e08215-dd40-40fe-bfcb-6ef135bbf6b9" />

After switching language to french
<img width="754" height="747" alt="Image" src="https://github.com/user-attachments/assets/2cff8bb2-766a-4728-b553-93bf117cd41c" />

After closing and reopening the settings window
<img width="751" height="746" alt="Image" src="https://github.com/user-attachments/assets/c74d79de-8029-4eec-9ac5-2825158e9291" />
